cmd/orb_import_test.go

```diff
@@ -808,7 +808,7 @@ The following orb versions already exist:
 				  }`
 
 			createOrbResp := `{
-					"createOrb": {
```

Comment:
~~ the var name `createOrbResp` could get changed to `importOrbResp`

Reply:
Thanks for the catch!